src/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const DEFAULT_VALUE = '-';
+export const columnNames = ['Name', 'Sex', 'Born', 'Died'];
+
+export const SEX_FILTER = {
+  All: null,
+  Male: 'm',
+  Female: 'f',
+};
```

Comment:
You have this object and in another file you have `const Sex` object, do you really need both? Choose one and make it `enum`, not just object.

Reply:
Good point

Reply:
> Good point

Not fixed, without any reply 👀